alb.tf

```diff
@@ -38,7 +38,7 @@ resource "aws_lb_target_group" "alb_target_group" {
 resource "aws_lb_listener" "lb_http_listener" {
   count             = var.create_alb ? 1 : 0
   load_balancer_arn = aws_lb.alb[0].id
-  port              = 14000
```

Comment:
why did you change this?

Reply:
Because we need two listeners one for http and one for https and we can't have the same port on different protocol